src/snapred/backend/service/SousChef.py

```diff
@@ -66,36 +67,44 @@ def prepRunConfig(self, runNumber: str) -> RunConfig:
     def prepCalibrantSample(self, calibrantSamplePath: str) -> CalibrantSamples:
         return self.dataFactoryService.getCalibrantSample(calibrantSamplePath)
 
-    def prepPixelGroup(self, ingredients: FarmFreshIngredients):
+    def prepFocusGroup(self, ingredients: FarmFreshIngredients) -> FocusGroup:
+        if os.path.isfile(ingredients.focusGroup.definition):
```

Comment:
This check should probs be abstracted to the data layer, otherwise this is a service that touches disk directly.